### PR TITLE
ERC-1456 - Address Metadata JSON Schema

### PR DESCRIPTION
**Pull request by pedrouid**
####  ERC-1456 - Address Metadata JSON Schema
A standard JSON specification for storing metadata associated to Ethereum addresses.

### CODE REVIEW

**Comment by pedrouid** [comment]
cc @Cygnusfear

File: EIPS/eip-1357.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+```
+eip: 1357
```
Comment (Arachnid) [inline EIPS/eip-1357.md]: Where is this number derived from?

**Comment by pedrouid** [comment]
Thanks for pointing it out @Arachnid! I've fixed the EIP number

File: EIPS/eip-1456.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+type: Standards Track
+category: ERC
+status: Draft
+created: 2018-09-17
+discussions-to: https://ethereum-magicians.org/t/erc-1456-address-metadata-json-schema/1491
+---
+
+## Address Metadata JSON Schema
+
+## Summary
```
Comment (axic) [inline EIPS/eip-1456.md]: Change this to "Simple Summary" please.

File: EIPS/eip-1456.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+title: Address Metadata JSON Schema
+authors: Alexander Mangel <alexander.mangel@protonmail.com>, Pedro Gomes <pedrouid@protonmail.com>
+type: Standards Track
+category: ERC
+status: Draft
+created: 2018-09-17
+discussions-to: https://ethereum-magicians.org/t/erc-1456-address-metadata-json-schema/1491
+---
+
+## Address Metadata JSON Schema
```
Comment (axic) [inline EIPS/eip-1456.md]: I'm not sure this is needed.

File: EIPS/eip-1456.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+Ethereum addresses have related information (metadata) useful to users and developers. This metadata is captured by third parties and not accessible through an on-chain mechanism.
+
+ETH Registry makes address metadata available to applications and users through a registry contract that maps the addres to a JSON file stored on IPFS.
+
+A working prototype is available at: https://ethregistry.org.
+
+## Specification
+
+Inspired by:
```
Comment (axic) [inline EIPS/eip-1456.md]: I think this list goes better into the Rationale.

File: EIPS/eip-1456.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+               "ponzi",
+               "binance"],
+        description: "",      // Further details
+        addresses: []         // Related addresses
+        related: [],          // Related JSON IPFS hashes
+    },
+}
+```
+
+## Standard Specific Metadata
```
Comment (axic) [inline EIPS/eip-1456.md]: I'm not sure what this section is?

**Comment by axic** [comment]
Ping @pedrouid

**Pull request closed without merging**